src/openrct2-ui/windows/NewRide.cpp

```diff
@@ -45,6 +46,7 @@ static constexpr const int32_t WindowWidth = 601;
 static constexpr const int32_t RideListItemsMax = 384;
 static constexpr const int32_t RideTabCount = 6;
 static constexpr const int32_t GroupByTrackTypeWidth = 172;
+static std::string Filter;
```

Comment:
For static variables like this one, we use the notation `_filter`.

Reply:
This has been changed